service_mesh/v2x/ossm-deployment-models.adoc

```diff
@@ -0,0 +1,14 @@
+[id="ossm-deployment-models"]
+= Deployment topology models
```

Comment:
This is the only instance of "topology" in the PR. Just wondering if it's necessary (or if so, if it should be elsewhere too)